6_QAQC/src/QAQC_Approach3_lat_long_doy_bins.R

```diff
@@ -0,0 +1,70 @@
+library(scipiper)
+library(dplyr)
+library(plotly)
+library(tidyverse)
+set.seed
+
+#getting the data from google drive and changing the file to RDS file without ind
+gd_get("5_data_munge/out/daily_temperatures.rds.ind")
+# to get the sites longitude and latitude data from google drive,
+gd_get("5_data_munge/out/all_sites.rds.ind")
+
+#creating an r object for the daily temp data
+daily_dat <- readRDS('5_data_munge/out/daily_temperatures.rds') %>%
+  filter(!is.na(date)) %>%
+  mutate(month = lubridate::month(date), 
+         doy = lubridate::yday(date))
+
+# aiming to find the 1 degree longitude and 1 degree latitude bins (interval) from the site data:
+# creating  bins (interval) using the cut function.
+# using the seq function and setting the interval starts with min of long/latitude
+#and ends with max of latitude, and each interval size is 1.
+
+long_deg <- 1
+lat_deg <- 1
+sites_dat <- readRDS("5_data_munge/out/all_sites.rds") %>%
+  filter(!is.na(latitude)) %>%
+  mutate(lat_bins = cut(latitude, breaks = 
+                          seq(from = floor(min(latitude)),
+                              to = ceiling(max(latitude)),
+                              by = lat_deg),
+                        right = FALSE),
+         long_bins = cut(longitude, breaks = 
+                           seq(from = floor(min(longitude)),
+                               to = ceiling(max(longitude)),
+                               by = long_deg), right = FALSE))
+
+# removing duplicated sites_data
+sites_dat_unique <- unique(sites_dat)
```

Comment:
You can remove this step, as the duplication will be handled elsewhere in the pipeline.